Source/WebCore/testing/Internals.cpp

```diff
@@ -3325,7 +3325,7 @@ ExceptionOr<ScrollableArea*> Internals::scrollableAreaForNode(Node* node) const
             return Exception { ExceptionCode::InvalidAccessError };
 
         auto& renderBox = *element.renderBox();
-        if (!renderBox.canBeScrolledAndHasScrollableArea())
+        if (!shouldIncludeNonscrollableAreas && !renderBox.canBeScrolledAndHasScrollableArea())
```

Comment:
I think we should remove the new argument and instead of `renderBox.canBeScrolledAndHasScrollableArea` just try to get the renderBox's scrollableArea.

Reply:
Fixed.